src/pathwayvis/components/tooltip/newTooltip.ts

```diff
@@ -0,0 +1,133 @@
+import * as tinier from 'tinier';
+import * as _ from 'underscore';
+
+function decompartmentalize(id) {
```

Comment:
There's no way to access these helpers form escher?

Reply:
Is this file copied over from escher? There was no way to extend the code coming form escher? If no, could you please highlight where you extended / modified the original code?